apps/api/src/routers/user.py

```diff
@@ -141,7 +141,7 @@ async def _apply_flow(
         uid=user.uid,
         first_name=raw_application_data.first_name,
         last_name=raw_application_data.last_name,
-        roles=(Role.APPLICANT, Role[raw_application_data.application_type]),
+        roles=(Role.APPLICANT, raw_application_data.application_type),  # type: ignore
```

Comment:
I think you can construct the enum object instead as `Role(application_type)` to satisfy the typing, although that assumes the key is valid (which it should be).